backend/remote/backend_test.go

```diff
@@ -16,6 +16,18 @@ func TestRemote(t *testing.T) {
 	var _ backend.CLI = New(nil)
 }
 
+func TestRemote_backendDefault(t *testing.T) {
+	b := testBackendDefault(t)
+	backend.TestBackendStates(t, b)
+	backend.TestBackendStateLocks(t, b, b)
+	backend.TestBackendStateForceUnlock(t, b, b)
+}
+
+func TestRemote_backendNoDefault(t *testing.T) {
+	b := testBackendNoDefault(t)
+	backend.TestBackendStates(t, b)
+}
+
```

Comment:
This doesn't look like a diff, just checking if there was supposed to be something other than moving the source.

Reply:
Nope, I just wanted to move these to the top of the page 👍